packages/mina-graphql/src/Providers/TxSubmit/mutations.ts

```diff
@@ -0,0 +1,133 @@
+export function getTxSend(isRawSignature: boolean): string {
+  if (isRawSignature) {
+    return `
+        mutation sendTx($fee:UInt64!, $amount:UInt64!,
+        $to: PublicKey!, $from: PublicKey!, $nonce:UInt32, $memo: String,
+        $validUntil: UInt32, $rawSignature: String!) {
+          sendPayment(
+            input: {
+              fee: $fee,
+              amount: $amount,
+              to: $to,
+              from: $from,
+              memo: $memo,
+              nonce: $nonce,
+              validUntil: $validUntil
+            }, 
+            signature: {rawSignature: $rawSignature}) {
+            payment {
```

Comment:
We can try using GraphQL's fragments for the repeating parts of queries and mutations I think.
https://blog.logrocket.com/graphql-fragments-explained/
For now it's good enough, but let's create a ticket for this debt to create reusable pieces.